packages/html_of_jsx/html_of_jsx.0.0.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Render HTML writing JSX"
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/davesnx/html_of_jsx"
+bug-reports: "https://github.com/davesnx/html_of_jsx/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.0.0"}
+  "reason" {>= "3.10.0"}
+  "ppxlib" {> "0.23.0" & <= "0.31.0"}
```

Comment:
```suggestion
  "ppxlib" {> "0.23.0"}
```
Is this a known incompatibility? Otherwise please do not add an upper bound since we will add it manually if an update of ppxlib breaks the package

Reply:
True, there' no issue